microsoft/botframework-solutions
# [Virtual Assistant][TypeScript] Adapt documentation to TypeScript solution

## Description

- Adapt related documentation to TypeScript solution
- Fix typos and minor changes

Related to #769 

## Testing Steps
1. Go to `AI/solutions/Virtual-Assistant/docs`
2. Check the new changes for each document

## Checklist
